### DISCUSSION TITLE
Paste command behavior

### DISCUSSION THREAD

**Discussion by ayuvlasenko** [JetBrains/ideavim#367] [General]
After latest update (1.7.0) paste (p) command behavior has changed. Now, if i copy 1st line, copy 2nd line and then press p 2 times, it pastes different lines (video attached). Original vim behaves differently, it pastes same line each time (plugin had same behavior before update). How can i return previous behavior?

https://user-images.githubusercontent.com/77232221/133756332-b99a5dd1-b5f1-498a-ad78-34e9ad112cb0.mp4

**Comment by AlexPl292** [comment]
Hi, the fix will be available in a hotfix release. Thank you for the report.
